starters/gatsby-wordpress-starter/__tests__/data/pages.json

```diff
@@ -1,26 +1,18 @@
-[
-	{
-		"previous": null,
-		"page": {
-			"id": "cG9zdDoxMA==",
-			"title": "Test Page",
-			"uri": "/test-page/",
-			"featuredImage": null
-		},
-		"next": { "id": "cG9zdDoy", "title": "Sample Page", "uri": "/sample-page/" }
-	},
-	{
-		"previous": {
-			"id": "cG9zdDoxMA==",
-			"title": "Test Page",
-			"uri": "/test-page/"
-		},
-		"page": {
-			"id": "cG9zdDoy",
-			"title": "Sample Page",
-			"uri": "/sample-page/",
-			"featuredImage": null
-		},
-		"next": null
+{
```

Comment:
This change seems strange to me - the data is not the same and also it's not in the same shape.